### PR TITLE
Enable SSR capability

### PR DESCRIPTION
**Pull request by haffla**
Hey @se-panfilov 
thanks for this library. Unfortunately it is not working in a server side rendering setup due to the use of `window.document` which of course is undefined outside of the browser. May I suggest this little change? The idea is to not run the code server side - it's only needed client side anyway.

### CODE REVIEW

File: dist/mini-toastr.es5.js [hunk 1]
```diff
@@ -19,6 +19,8 @@ var miniToastr = function () {
 
   var _ref, _style;
 
+  if (typeof window === 'undefined') return;
```
Comment (se-panfilov) [inline dist/mini-toastr.es5.js]: __[eslint]__
* Error - Extra semicolon. (semi)

File: dist/mini-toastr.es5.min.js [hunk 2]
```diff
@@ -1,2 +1,2 @@
...
...E"}),e(a,"&."+p.info,{"background-color":"#4196E1"}),e(a,"&:hover",{opacity:1,"box-shadow":"#000 0 0 12px"}),a)),e(c,"."+p.title,{"font-weight":"500"}),e(c,"."+p.message,{display:"inline-block","vertical-align":"middle",width:"240px",padding:"0 12px"}),c)},u={config:l,showMessage:function(e,t,n,o,i,r){var a={};Object.assign(a,this.config),Object.assign(a,r);var c=document.createElement("div");if(c.className=p.notification+" "+p[n],c.onclick=function(){a.animation(c,null)},t){var s=document.createElement("div");s.className=p.title,s.appendChild(document.createTextNode(t)),c.appendChild(s)}if(e){var d=document.createElement("div");d.className=p.message,d.appendChild(document.createTextNode(e)),c.appendChild(d)}return a.node.insertBefore(c,a.node.firstChild),setTimeout(function(){return a.animation(c,i)},o||a.timeout),i&&i(),this},init:function(e){var t=this,n={};Object.assign(n,l),Object.assign(n,e);var o=i(n.style);return r(o),n.node.id=""+p.container,n.node.className=""+p.container,n.appendTarget.appendChild(n.node),Object.keys(n.types).forEach(function(e){u[n.types[e]]=function(t,o,i,r,a){return this.showMessage(t,o,n.types[e],i,r,a),this}.bind(t)}),this}};return u}}();return n});
```
Comment (se-panfilov) [inline dist/mini-toastr.es5.min.js]: __[eslint]__
* Error - Wrap an immediate function invocation in parentheses. (wrap-iife)
* Error - Missing space before function parentheses. (space-before-function-paren)
* Error - A space is required after '
* Error - Missing space before opening brace. (space-before-blocks)
* Error - Requires a space after '{'. (block-spacing)
* Error - Strings must use singlequote. (quotes)
* Error - Expected literal to be on the right side of ==. (yoda)
* Error - Infix operators must be spaced. (space-infix-ops)
* Error - Expected '===' and instead saw '=='. (eqeqeq)
* Error - 'define' is not defined. (no-undef)
* Error - Requires a space before '}'. (block-spacing)
* Error - Missing whitespace after semicolon. (semi-spacing)
* Error - Return statement should not contain assignment. (no-return-assign)
* Error - Missing space before value for key 'value'. (key-spacing)
* Error - Missing space before value for key 'enumerable'. (key-spacing)
* Error - Missing space before value for key 'configurable'. (key-spacing)
* Error - Missing space before value for key 'writable'. (key-spacing)
* Error - Unexpected use of comma operator. (no-sequences)
* Error - Split initialized 'var' declarations into multiple statements. (one-var)
* Error - Expected space(s) before "var". (keyword-spacing)
* Error - A leading decimal point can be confused with a dot. (no-floating-decimal)
* Error - Expected space(s) before "function". (keyword-spacing)
* Error - Expected space(s) after "for". (keyword-spacing)
* Error - Expected space(s) before "if". (keyword-spacing)
* Error - Expected space(s) after "if". (keyword-spacing)
* Error - Expected literal to be on the right side of ===. (yoda)
* Error - Expected space(s) before "return". (keyword-spacing)
* Error - Expected literal to be on the right side of !=. (yoda)
* Error - Expected '!==' and instead saw '!='. (eqeqeq)
* Error - Missing space before value for key 'error'. (key-spacing)
* Error - Missing space before value for key 'warn'. (key-spacing)
* Error - Missing space before value for key 'success'. (key-spacing)
* Error - Missing space before value for key 'info'. (key-spacing)
* Error - Missing space before value for key 'container'. (key-spacing)
* Error - Missing space before value for key 'notification'. (key-spacing)
* Error - Missing space before value for key 'title'. (key-spacing)
* Error - Missing space before value for key 'message'. (key-spacing)
* Error - Missing space before value for key 'types'. (key-spacing)
* Error - Missing space before value for key 'animation'. (key-spacing)
* Error - Missing space before value for key 'timeout'. (key-spacing)
* Error - Missing space before value for key 'appendTarget'. (key-spacing)
* Error - Missing space before value for key 'node'. (key-spacing)
* Error - Missing space before value for key 'style'. (key-spacing)
* Error - Missing space before value for key 'position'. (key-spacing)
* Error - Missing space before value for key 'z-index'. (key-spacing)
* Error - Missing space before value for key 'right'. (key-spacing)
* Error - Missing space before value for key 'top'. (key-spacing)
* Error - Missing space before value for key 'cursor'. (key-spacing)
* Error - Missing space before value for key 'padding'. (key-spacing)
* Error - Missing space before value for key 'margin'. (key-spacing)
* Error - Missing space before value for key 'background-color'. (key-spacing)
* Error - Missing space before value for key 'opacity'. (key-spacing)
* Error - Missing space before value for key 'color'. (key-spacing)
* Error - Missing space before value for key 'border-radius'. (key-spacing)
* Error - Missing space before value for key 'box-shadow'. (key-spacing)
* Error - Missing space before value for key 'width'. (key-spacing)
* Error - Missing space before value for key 'font-weight'. (key-spacing)
* Error - Missing space before value for key 'display'. (key-spacing)
* Error - Missing space before value for key 'vertical-align'. (key-spacing)
* Error - Missing space before value for key 'config'. (key-spacing)
* Error - Missing space before value for key 'showMessage'. (key-spacing)
* Error - Missing space before value for key 'init'. (key-spacing)
* Error - Extra semicolon. (semi)

File: dist/mini-toastr.es6.js [hunk 3]
```diff
@@ -2,6 +2,8 @@
 var miniToastr = (function () {
   'use strict'
 
+  if(typeof window === 'undefined') return
```
Comment (se-panfilov) [inline dist/mini-toastr.es6.js]: __[eslint]__
* Error - Expected space(s) after "if". (keyword-spacing)

File: src/main.js [hunk 4]
```diff
@@ -2,6 +2,8 @@
 var miniToastr = (function () {
   'use strict'
 
+  if(typeof window === 'undefined') return
```
Comment (se-panfilov) [inline src/main.js]: __[eslint]__
* Error - Expected space(s) after "if". (keyword-spacing)

**Comment by se-panfilov** [comment]
Click here if you want to create eslint's auto-correct Pull Request
[Auto Correct by SideCI](https://sideci.com/gh/70707233/issues/pull_request/79/diff/65ae16139bdd2613a932be32a1fdd7c3141a3e74?open_modal=1#eslint)

See the documentation if you want to get more information about auto-correct.
[Document](https://sideci.com/ja/docs/autofix-eslint)

File: dist/mini-toastr.es5.min.js (hunk 2 above)
Comment (haffla) [inline dist/mini-toastr.es5.min.js]: eslint on minified dist files?

**Comment by haffla** [comment]
gulp-connect is on the latest version (5.0.0) but bitHound marks it as deprecated because the latest release is from 7 months ago, oh boy

**Comment by se-panfilov** [comment]
Hi @haffla 

Thanks for PR!
I'll take a look shortly and fix this stuff by myself

**Comment by se-panfilov** [comment]
@haffla 
Meet v0.5.0 =)

Btw, added `dist` folder into `.eslintignore`

**Pull request merged**